tests/test_workers/test_celery_logging.py

```diff
@@ -1,28 +1,71 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import logging
+import os
+import pytest
```

Comment:
Could you please move this to the second section of imports since this is a third-party library?